jujugui/static/gui/src/app/components/date-display/date-display.js

```diff
@@ -0,0 +1,145 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2016 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('date-display', function() {
+
+  juju.components.DateDisplay = React.createClass({
+    // Set the current date on the component so that it can be overridden in
+    // tests.
+    NOW: new Date(),
```

Comment:
I believe that this will create a new Date at the time the component is created not every time it's rendered.